coredocument/coredocument.proto

```diff
@@ -73,7 +73,8 @@ message CoreDocument {
   // Collaborators involved in the document consensus
   repeated bytes collaborators = 17;
   // list of roles
-  repeated RoleEntry roles = 1 [
+  repeated Role roles = 1 [
+    (proofs.key_length) = 32,
```

Comment:
which key is this key_length property referring to?

Reply:
This refers to the length of the mapping_key below. As we are compressing the properties when generating trees/proofs there is need to know how long is that value so it can be parsed correctly out of the resulting array of bytes.